opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -776,6 +776,9 @@ namespace Opm {
 
         for (int w = 0; w < nw; ++w) {
             const bool is_producer = well_container_[w]->wellType() == PRODUCER;
+            const int well_cell_top = well_container_[w]->cells()[0];
+#warning HACK
+            const int pvtRegionIdx = 0;
 
```

Comment:
@GitPaean I either need to store the cellPvtIdx map in StandardWellsDense or store a pvtnumIdx pr well. Do you have any preference. Do you know of pvtIdx could be of interest in other part of the well model.

Reply:
If we assume the well is only completed in one PVT region, it makes sense to store a pvt region number for each well, you can put it in WellInterface. 
I remember there is some kind of rule to determine the PVT region for a well, which is based on the lowest (bottommost) perforated cell, while I could not find it now.  But in case you encounter problem in testing, you can also try this option.  

I am having problem in determining the PVT region for a segment. I am using a similar approach with this PR for now. Until we have a case the PVT region of the well / segment is a problem.